crates/pop-parachains/src/templates.rs

```diff
@@ -233,8 +232,8 @@ mod tests {
 			("evm".to_string(), EVM),
 			// openzeppelin
 			("polkadot-generic-runtime-template".to_string(), OpenZeppelinGeneric),
+			("polkadot-sdk-parachain-template".to_string(), ParityGeneric),
```

Comment:
Template names are getting confusing - they should have been prefixed with provider name. Perhaps we should check the stats to see if anyone is even using these identifiers and potentially deprecate?

An alternative might be using the GH org/repo convention or something.

Adding a new template is an opportunity to improve.

Reply:
I agree that using the command line might be confusing for some users, when prompting the UI is more clear than in the command line. Are you suggesting to use `paritytech/polkadot-sdk-parachain-template`?

Reply:
Perhaps. Its rather long tho, but how many people are gonna type it? Some structure is worth considering but any decision guided by data.

Reply:
@brunopgalvao, what are your thoughts based on your experience testing it?

Reply:
Something like what Alex proposes makes sense to me as well. `prov/template-name` is unique enough to identify each template without ambiguity.
There are certain parts of the code that use the serialized name of the template like `template.to_string()`, that should be reviewed as we introduced some structure in the naming.
We could implement a function for `Parachain` too so that we can strip the `prov/` part of it and use it as needed.

Reply:
Yes, I'll handle this in the other https://github.com/r0gue-io/pop-cli/pull/284
The naming here is as Frank suggested: paritytech/polkadot-sdk-parachain-template so use the same style for Tanssi templates adding the prefix